README.md

```diff
@@ -1,8 +1,14 @@
-Implementation of a persistence layer using JDBC API for an Airlines Application.
+### Implementation of a persistence layer using JDBC API for an Airlines Application.
+***
+#### Prerequisites:
+- System should have installed PostgreSQL
 
-Prerequisites:
-System should have installed PostgreSQL
-Install notes:
-1 Create database airlines
-2 Execute schema.sql script
-3. Execute data.sql script for mock data (optional)
\ No newline at end of file
+#### Install notes:
+1. Create database and corresponding user:
+   CREATE USER airlines;
+   ALTER USER airlines WITH PASSWORD 'airlines';
```

Comment:
README.md old state appearance:
![image](https://user-images.githubusercontent.com/82450396/115423679-b54a9700-a206-11eb-8ccc-2713c82daf2c.png)

README.md raw file:
![image](https://user-images.githubusercontent.com/82450396/115423914-e0cd8180-a206-11eb-81db-a0a1e08f5aa1.png)

You deleted code block symbols and now it appears as:
![image](https://user-images.githubusercontent.com/82450396/115424058-ffcc1380-a206-11eb-8860-68f693fc4fd0.png)

Put your changes into code block that marks by three ` symbols